testing/py-avro/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Maintainer: Gennady Feldman <gena01@gmail.com>
+# Contributor: Gennady Feldman <gena01@gmail.com>
+pkgname=py3-avro
+_pkgname=avro-python3
+pkgver=1.9.0
+pkgrel=0
+pkgdesc="Avro is a serialization and RPC framework."
+url="https://avro.apache.org/"
+arch="noarch"
+license="Apache-2.0"
+depends="py3-six"
+makedepends="py3-setuptools python3-dev"
+source="$pkgname-$pkgver.tar.gz::http://apache.mirrors.tds.net/avro/stable/py3/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	python3 setup.py build
+}
```

Comment:
missing check()

Reply:
There are no tests? in the archive file?

Reply:
Logged a ticket: https://issues.apache.org/jira/browse/AVRO-2420